### PR TITLE
Tim/#62/upload image

### LINKED ISSUE #62: app registration pages
Create app registration pages:

Image

Ensure you include the following:

- [ ] Full app feature
- [ ] At least 50% UI test coverage. (using jest)
- [ ] Add appropriate routing
- [ ] Refactor backend API mutations to match UI design structure.
- [ ] Refactor `user` DB schema to factor in the reason for registration.
- [ ] Refactor DB schema to accommodate image upload for a user account using `ActiveStorage`.
- [ ] Add a feature for uploading Images. (**_Make this feature re-usable for other parts of the app_**)
- [ ] Ensure the uploading image feature has at least 75% test coverage.

### PR DESCRIPTION
**Pull request by timothykiyeng**
This pull request handles:
Photo upload using ActiveRecord
Testing for photo upload functionality

### CODE REVIEW

**Comment by timothykiyeng** [comment]
Gem apollo upload server:         https://github.com/jetruby/apollo_upload_server-ruby

File: api/app/models/user.rb [hunk 1]
```diff
@@ -12,13 +13,29 @@ class User < ApplicationRecord
...
+      end
+    end
+
+    def avatar_attached?
+      avatar.attached?
+    end
+
+    def avatar_validations
+      if avatar.attached?
+        unless ['image/png', 'image/jpeg'].include?(avatar.blob.content_type)
```
Comment (otsembo) [inline api/app/models/user.rb]: We can also allow for `.jpg` format

**Review by otsembo — APPROVED** [review]
Hello @timothykiyeng Great job on this PR. Everything looks good and works perfectly.
Just fix the one minor issue and merge the PR.

File: api/app/models/user.rb (hunk 1 above)
Comment (timothykiyeng) [reply api/app/models/user.rb]: All right. Noted.

**Pull request merged**